test/node/conversation-archived.js

```diff
@@ -13,19 +13,14 @@ describe('Conversation Archived', () => {
     before(async () => {
         client = new Circuit.Client(config.bot1);
         user = await client.logon();
+        const topic = `${Date.now()}a`;
+        conversation = await client.createConferenceBridge(topic);
```

Comment:
Could you use the preparation conversation here?

Reply:
done